internal/enforcer/enforcer.go

```diff
@@ -87,6 +101,39 @@ func (c *Conform) Enforce(setters ...policy.Option) {
 	}
 }
 
+// SetStatus sets the status of a GitHub check.
+// Valid statuses are "error", "failure", "pending", "success"
+func (c *Conform) SetStatus(state, policy, check, message string) {
+	if c.token == "" {
```

Comment:
This is a cheesy way to make this optional for now. We have more immediate use cases and I can circle back and refactor. I'm thinking of a `Summarizer` interface that can be `GitHub, PlainText,etc.`.